src/widgets/auth/views/signupViewComponent.jsx

```diff
@@ -71,6 +96,10 @@ export default class SignupView extends React.Component {
             ]
             : fields;
 
+        const sharedProps = {
+            blacklist: this.state.blacklist
+        }
```

Comment:
Reinventing Redux? 😉

Reply:
I'm not super familiar with Redux. Do you see another solution?